docs/management/service-discovery.md

```diff
@@ -0,0 +1,69 @@
+---
+title: "Service Discovery"
+---
+
+Convox makes it easy to connect your services.
+
+#### Linking Services Within a Single App
+
+If you have multiple services defined in your application you can link those services by defining a `links` section in your [convox.yml](/application/convox-yml).
+For example if you have a service called `web` and a service called `api` and you would like `web` to be able to call `api` your convox.yml might look like:
+
+```
+services:
+  web:
+    build: .
+    port: 3000
+    resources:
+      - database
+    links:
+      - api
+  api:
+    build: .
+    port: 3000
+```
+
+This will cause Convox to automatically inject an environment variable called `API_URL` in all containers running the `web` service.
+
+#### Communicating Between Apps 
+
+There is not currently a process for automated discovery of services between apps. That said, it is a relatively simple process to connect services across apps. In order to find the endpoint for your service simply run `convox services`
```

Comment:
`servicename.appname.rackname.convox` works internally on the VPC

Reply:
@ddollar  Yeah I was debating whether to include this or not. I had always thought of this as a bit of a "hack" that we might not support forever but if you think it's the official way to communicate between apps, as opposed to using the url from convox services, I am happy to include it.